tests/erlang_tests/test_process_info.erl

```diff
@@ -28,6 +28,11 @@ start() ->
     receive
         ok -> ok
     end,
+    [] = erlang:process_info(Self, registered_name),
+    erlang:register(has_name, Self),
+    {registered_name, Name} = erlang:process_info(Self, registered_name),
+    assert(Name =:= has_name),
+    erlang:unregister(has_name),
     test_message_queue_len(Pid, Self),
```

Comment:
Can we duplicate line 31 here to
check that `registered_name` is no longer available?